### PR TITLE
data: khushbumaheshwarii

### PR DESCRIPTION
**Pull request by khushbumaheshwarii**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6186"></a>

### CODE REVIEW

**Review by shyamtawli — CHANGES_REQUESTED** [review]
add your json file to `data/` folder.

File: khushbumaheshwarii.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+	{
+  "name": "Khushbu Maheshwari",
+  "displayStarsPublic":true,
+  "type": "personal",
+  "bio": "Student | frontend developer | Exploring **Open Source**",
+  "avaar":"https://github.com/khushbumaheshwarii.png"
```
Comment (shyamtawli) [inline khushbumaheshwarii.json:6]: you can remove this line.
```suggestion
 
```

**Review by shyamtawli — CHANGES_REQUESTED** [review]

File: data/khushbumaheshwarii.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+	{
+  "name": "Khushbu Maheshwari",
+  "displayStarsPublic":true,
```
Comment (harshsingh32) [inline data/khushbumaheshwarii.json:3]: please remove "displayStarsPublic":true

**Comment by eddiejaoude** [comment]
Hi @khushbumaheshwarii did you mean to close your pull request?

File: data/khushbumaheshwarii.json (hunk 2 above)
Comment (eddiejaoude) [reply data/khushbumaheshwarii.json:3]: Thank you for reviewing @harshsingh32 but this comment was already made earlier [here](https://github.com/EddieHubCommunity/LinkFree/pull/6186#pullrequestreview-1390319158), repeating comments does not help, it is spam and scares people off from contributing, please read existing comments on the timeline first

Comment (harshsingh32) [reply data/khushbumaheshwarii.json:3]: I'll read existing comments on the timeline first.

**Pull request closed without merging**